src/ResourceTemplate/AbsoluteResourceTemplate.php

```diff
@@ -61,10 +61,10 @@ class AbsoluteResourceTemplate implements ResourceTemplateInterface
 
     /**
      * AbsoluteResourceTemplate constructor.
-     * @param string $path
+     * @param string|null $path
```

Comment:
So why `string|null` here, but not elsewhere? There are a few other spots that have a `string` param that is optional and defaults to `null`.

Reply:
I think you're right, and the optional `null` should be removed.